CHANGELOG.md

```diff
@@ -6,11 +6,16 @@
 * Adds missing `license_number` option
 * Adds missing `receiver_liquor_license` option
 * Adds missing `VerificationDetails` object (closes #140, #141, #162)
-* Bumps `RestSharp` from `106.4.2` to `106.12.0`
 * Changes the Client delegate constructor from `internal` to `public`
 * Updates various information in the README related to thread-safety, examples, releasing, etc
 * Adds all missing dates and versions to the CHANGELOG
 
+## 2.5.1.4 (2021-11-03)
+
+* Bumps `RestSharp` from `106.12.0` to `106.13.0`
```

Comment:
This line should read `Bumps RestSharp from 106.4.2 to 106.13.0` since it was bumped from 4.2 to 13 since our last real release.

Reply:
Updated